DEVELOPING_OPAL/tools/src/main/scala/org/opalj/support/info/Purity.scala

```diff
@@ -479,235 +577,60 @@ object Purity {
-        var callGraphName: Option[String] = None
-        var configurationName: Option[String] = None
-        var schedulingStrategy: Option[String] = None
-        var withoutJDK = false
-        var individual = false
-        var isLibrary = false
-        var cwa = false
-        var debug = false
-        var multiProjects = false
-        var eager = false
-        var evaluationDir: Option[File] = None
-        var packages: Option[Array[String]] = None
-        var numThreads = PropertyStoreKey.parallelismLevel
-
-        // PARSING PARAMETERS
-        var i = 0
-
-        def readNextArg(): String = {
-            i += 1
-            if (i < args.length) {
-                args(i)
-            } else {
-                println(usage)
-                throw new IllegalArgumentException(s"missing argument: ${args(i - 1)}")
-            }
-        }
+        val purityConf = new PurityConf(args)
 
-        while (i < args.length) {
-            args(i) match {
-                case "-cp"                 => cp = new File(readNextArg())
-                case "-projectDir"         => projectDir = Some(readNextArg())
-                case "-libDir"             => libDir = Some(readNextArg())
-                case "-analysis"           => analysisName = Some(readNextArg())
-                case "-fieldAssignability" => fieldAssignabilityAnalysisName = Some(readNextArg())
-                case "-escape"             => escapeAnalysisName = Some(readNextArg())
-                case "-domain"             => domainName = Some(readNextArg())
-                case "-rater"              => raterName = Some(readNextArg())
-                case "-callGraph"          => callGraphName = Some(readNextArg())
-                case "-analysisName"       => configurationName = Some(readNextArg())
-                case "-schedulingStrategy" => schedulingStrategy = Some(readNextArg())
-                case "-eager"              => eager = true
-                case "-individual"         => individual = true
-                case "-closedWorld"        => cwa = true
-                case "-library"            => isLibrary = true
-                case "-debug"              => debug = true
-                case "-multi"              => multiProjects = true
-                case "-eval"               => evaluationDir = Some(new File(readNextArg()))
-                case "-packages"           => packages = Some(readNextArg().split(':'))
-                case "-j"                  => numThreads = readNextArg().toInt
-                case "-noJDK"              => withoutJDK = true
-                case "-JDK" =>
-                    cp = JRELibraryFolder; withoutJDK = true
-
-                case unknown =>
-                    Console.println(usage)
-                    throw new IllegalArgumentException(s"unknown parameter: $unknown")
-            }
-            i += 1
+        if(args.contains("--help") || args.contains("-h")) {
```

Comment:
Can this be made less of a boilerplate? E.g., asking the conf object whether help was asked for?